packages/react-native-sdk/docusaurus/docs/reactnative/05-ui-cookbook/10-connection-quality-indicator.mdx

```diff
@@ -3,7 +3,88 @@ id: connection-quality-indicator
 title: Connection Quality Indicator
 ---
 
-:::info
-This guide is still in progress and will be updated soon.
-If you have any questions, please reach out to us on our [website](https://getstream.io/video/#contact).
-:::
+Network conditions are not always perfect so instead of playing guessing games as to why is someone's video so blurry our default [`ParticipantView`](../../ui-components/participants/participant-view/) component comes equipped with connection quality indicator which provides general information about that participants network conditions in four states:
+
+- UNSPECIFIED
+- POOR
+- GOOD
+- EXCELLENT
+
+![Preview of the Connection quality indicator](../assets/06-advanced/10-connection-quality-indicator/connection-quality-indicator.png)
+
```

Comment:
small nitpick on the patch, it should not be inside 06-advanced but it should be inside 05-ui-cookbook

very nice screenshot though